### ISSUE TITLE
SERVER

### ISSUE THREAD

**Issue by shinchan14** [Coopdis/easyMesh#2]
in each mesh we have a server
whether for server we can make use of library easywebserver or is embedded in the feature easymesh?
thanks

**Comment by Coopdis** [comment]
Yes.  So the easyMesh protocol runs on top of TCP, so each node establishes a TCP server that is able to take connection requests from other nodes.  In the startHere example, this port is 5555 but it is entirely configurable.

There is no web server or web socket server that easyMesh is dependent on.  These services are up to the application program to include.

For my applications, web servers and web sockets servers have been critical, so I have also written libraries that provide those functions, but they are not required for easyMesh to work properly... as is seen in the startHere example.

The reason that a wrote these libraries and didn't just use the stock Arduino libraries is that I found the stock Arduino libraries to be rather slow and laggy.  I needed better performance so I reimplemented those functions using the native esp8266 SDK libraries.  They are much faster and more responsive... at least in my experience.

Best,

Bill
